packages/components/src/stores/pharmacy.ts

```diff
@@ -65,11 +65,14 @@ export const createPharmacyStore = (cs?: typeof createStore) => {
   const getPharmaciesByAddress = async (
     client: PhotonClient,
     geocoder: google.maps.Geocoder,
-    address: string,
+    address: string | { lat: number; lng: number },
```

Comment:
hmm, wonder if we want to split these 🤔, address makes me think street address not lat/lng

Reply:
gonna come back to this, I have a todo to rethink how the data is handled with store